### PR TITLE
Frontendcomponents

### PR DESCRIPTION
**Pull request by victoramontoya**
trip list works and it renders cards. Created dummy trips. I think we need Redux.

### CODE REVIEW

File: client/src/components/Trips/TripView.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react"
+import { Link } from "react-router-dom"
+// import DeleteModal from "../Modal"
+import axios from "axios"
+
+class TripView extends Component {
```
Comment (iamthuypham) [inline client/src/components/Trips/TripView.js]: I suggest building `TripView` in your next pr. Save your code somewhere, and empty the component.

File: client/src/App.js [hunk 2]
```diff
@@ -11,7 +13,7 @@ const App = () => {
           <Route path="/" exact component={Home} />
           <Route path="/login" exact component={LogIn} />
           <Route path="/signup" exact component={SignUp} />
-          <Route path="/trips" exact component={TripsView} />
+          <Route path="/trips" exact component={TripList} />
```
[the remark this answers is not in the document]
Comment (iamthuypham) [reply client/src/App.js]: *** Hold this review for now ***
There is a discussion on Slack about MANAGING STATE: https://lambdaschoolstudents.slack.com/archives/GEM0G409F/p1544902559046000

File: client/src/components/Wrapper/Modal.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import { Button, Modal, ModalBody, ModalFooter } from "reactstrap"
+
+class DeleteModal extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      trip: {
+        id: this.props.trip.id
+      }
+    }
```
[the remark this answers is not in the document]
Comment (victoramontoya) [reply client/src/components/Wrapper/Modal.js]: wait I think I deleted the Modal

[the remark this answers is not in the document]
Comment (victoramontoya) [reply client/src/components/Wrapper/Modal.js]: Deleted modal component for now

**Pull request closed without merging**